.github/workflows/playwright.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     timeout-minutes: 60
     runs-on: ubuntu-latest
     container:
-      image: mcr.microsoft.com/playwright:v1.46.0-jammy 
+      image: mcr.microsoft.com/playwright:v1.48.0-jammy
```

Comment:
did you update this just in case?

Reply:
yes, stale references/updates to the lockfile required updates to the playwright version